Microsoft.Toolkit.Uwp.SampleApp/Pages/About.xaml

```diff
@@ -154,7 +154,7 @@
                             <TextBlock Margin="0,20,0,0"
                                        FontSize="14"
                                        Foreground="{ThemeResource Brush-Link-Normal}"
-                                       Text="The Windows Community Toolkit is a collection of helpers, extensions, and custom controls. It simplifies and demonstrates common developer tasks for building UWP and .NET apps for Windows 10."
+                                       Text="The Windows Community Toolkit is a collection of helpers, extensions, and custom controls. It simplifies and demonstrates common developer tasks for building UWP and WinUI 3 .NET apps for Windows 10 and Windows 11."
```

Comment:
Previous language implied `UWP`, `.NET`. The new language implies `UWP` and `WinUI 3 .NET`.

Did you mean `UWP`, `WinUI 3` and `.NET`? Or just UWP and WinUI 3?

Reply:
I want UWP .NET and WinUI 3 .NET effectively, so I shortened to not duplicate. I can go like this maybe

```suggestion
                                       Text="The Windows Community Toolkit is a collection of helpers, extensions, and custom controls. It simplifies and demonstrates common .NET developer tasks for building UWP and WinUI 3 apps for Windows 10 and Windows 11."
```

Reply:
I see! I think the suggestion looks good 🙂

Reply:
Tamam